option optimize_for = LITE_RUNTIME;

message CrusadeFortressFightRequest
{
 required uint32 id = 1;
}
